core/tools/coverage-report

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+COVERAGE_THRESHOLD=${COVERAGE_THRESHOLD:-0}
+
+coverage run --source=./src /dev/null 2>/dev/null
+mv .coverage .coverage.empty
+coverage combine .coverage.*
+
+# FIXME: fail-under may be useless because when a job fails no artifacts are produced
+# and the developer won't see the coverage report
```

Comment:
This is not true anymore?

Reply:
Could you actually confirm that the job actually fails if the covarage drops?

Reply:
Oops, leftover.

Example of the failing job: https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/691783206